README.md

```diff
@@ -16,6 +16,19 @@
  * specific language governing permissions and limitations
  * under the License.
 -->
+# cordova-weinre
+
+## Deprecation Notice
+
+This project is being deprecated. No more work will be done on this project by the Cordova development community. You can continue to use this project and it should work as-is, but issues will not be fixed by the Cordova community.
+
+weinre was built in an age when there were no remote debuggers available for mobile devices. Since then, some platforms are starting to provide remote debugger capabilities, as part of their platform tool set.
+
+1. Android:	[Get Started with Remote Debugging Android Devicese](https://developers.google.com/web/tools/chrome-devtools/remote-debugging/)
```

Comment:
Minor typo here, `Devicese` should be `Devices`

Reply:
thanks!